templates/build_script_python.sh

```diff
@@ -30,65 +32,65 @@ pip3 install -r /home/tester/output/requirements.txt
 pip3 freeze > /home/tester/output/available_packages.txt
 
 PACKAGE_INFO=`cat available_packages.txt | grep $PACKAGE_NAME`
```

Comment:
Should it be :
```
PACKAGE_INFO=`cat /home/tester/output/available_packages.txt | grep $PACKAGE_NAME`
```
instead of
```
PACKAGE_INFO=`cat available_packages.txt | grep $PACKAGE_NAME`
```

Reply:
Made suggested change